src/fshtypes/rules/ContainsRule.ts

```diff
@@ -11,6 +12,24 @@ export class ContainsRule extends Rule {
   get constructorName() {
     return 'ContainsRule';
   }
+
+  toFSH(): string {
+    const itemsWithAssociatedRules = this.items.map(item => {
+      let line = '';
+
+      // Add contains rule info
+      if (item.type) {
+        line += `${item.type} named ${item.name}`;
+      } else {
+        line += `${item.name}`;
+      }
+      return line;
+    });
+
+    return `* ${this.path} contains${
+      itemsWithAssociatedRules.length > 1 ? `${EOL}    ` : ' '
+    }${itemsWithAssociatedRules.join(` and${EOL}    `)}`; // New line and indent each
+  }
```

Comment:
I think this ends up writing syntactically invalid `ContainsRules` because a `ContainsRule` is always supposed to be followed by a cardinality.

Reply:
That's a good point. The FSH produced here is not generally intended to be run, but it should still be syntactically valid. Is it safe to always include a cardinality of `0..` as part of the `ContainsRule` with the expectation that a later `CardRule` will get the correct cardinality on to the slice? I feel like that's safe.

Reply:
I also feel like that's safe, good idea!